pkg/deploy/defaults.go

```diff
@@ -279,6 +288,16 @@ func GetCheMultiUser(cr *orgv1.CheCluster) string {
 	return DefaultCheMultiUser
 }
 
+func GetSingleHostExposureType(cr *orgv1.CheCluster) string {
+	dflt := DefaultKubernetesSingleHostExposureType
```

Comment:
`dflt` please no :D

Reply:
I simplified the logic such that the var is no longer ncssry. I stnd by dflt though :)